nautobot/extras/tests/test_forms.py

```diff
@@ -458,3 +463,111 @@ def test_update_relationship_associatioins_invalid_1(self):
             "Object Device 1 cannot form a relationship to itself!",
             form.errors[f"cr_{self.relationship_3.slug}__peer"][0],
         )
+
+
+class WebhookFormTestCase(TestCase):
+    @classmethod
+    def setUpTestData(cls):
+        obj_type_1 = ContentType.objects.get_for_model(ConsolePort)
+        obj_type_2 = ContentType.objects.get_for_model(Site)
```

Comment:
Can we make these more specific throughout?

```suggestion
        console_port_content_type = ContentType.objects.get_for_model(dcim_models.ConsolePort)
        site_content_type = ContentType.objects.get_for_model(dcim_models.Site)
```